packages/framework/src/Model/Cart/Cart.php

```diff
@@ -162,14 +162,14 @@ public function getItemById($itemId)
     /**
      * @return \Shopsys\FrameworkBundle\Model\Order\Item\QuantifiedProduct[]
      */
-    public function getQuantifiedProductsIndexedByItemId()
+    public function getQuantifiedProducts()
```

Comment:
I'm unfortunately unable to review whether you've changed everything that used to use cartIds